app/assets/javascripts/Components/Modals/graders_distribution_modal.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import Modal from "react-modal";
+import PropTypes from "prop-types";
+
+export class GraderDistributionModal extends React.Component {
+  static defaultProps = {
+    override: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.input = React.createRef();
+  }
+
+  componentDidMount() {
+    Modal.setAppElement("body");
+  }
+
+  onSubmit = event => {
+    event.preventDefault();
+    const weightings = {};
+    const form = new FormData(this.input.current);
+    form.forEach((key, value) => {
```

Comment:
Instead of this loop, you should be able to use `Object.fromEntries` (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries)